tests/compile-fail/methods.rs

```diff
@@ -238,6 +239,12 @@ fn or_fun_call() {
     //~^ERROR use of `unwrap_or`
     //~|HELP try this
     //~|SUGGESTION without_default.unwrap_or_else(Foo::new);
+
+    let mut btree = HashMap::<u64, String>::new();
```

Comment:
`btree = HashMap` looks a little insincere here. Did you want to test both and mixed up copy&paste?